tests/Data/Entries/EntryTest.php

```diff
@@ -74,9 +74,17 @@ public function it_sets_and_gets_the_slug()
     /** @test */
     public function the_slug_gets_slugified()
     {
+        Facades\Site::setConfig(['default' => 'en', 'sites' => [
+            'en' => ['locale' => 'en_US', 'url' => '/'],
+            'da' => ['locale' => 'da_DA', 'url' => '/da/'],
```

Comment:
`da_DK` ;)

Reply:
I'll update it, but it's completely irrelevant for this test.